docs/src/getting-started/checkpoints.rst

```diff
@@ -47,7 +47,8 @@ a file path.
     mtt export https://my.url.com/model.ckpt --output model.pt
 
 Downloading private HuggingFace models is also supported, by specifying the
-corresponding API token with the ``--huggingface_api_token`` flag.
+corresponding API token with the ``--huggingface_api_token`` flag or the
+``HUGGINGFACE_METATRAIN_TOKEN`` environment variable.
```

Comment:
I initially suggested changing the name, so the variable used for tests did not interfere with other use cases, but here I feel like this might be a use case where someone may want to use the same HF token for multiple projects. Is `HUGGINGFACE_TOKEN` or `HF_TOKEN` used by other projects? 

If we want to have a specific `HUGGINGFACE_METATRAIN_TOKEN`, I would suggest renaming the one in the tests to `HUGGINGFACE_METATRAIN_TOKEN_FOR_TESTS` or something similar.

TL;DR: I think it make sense to have a separate variable for tests and "normal" HF usage, which should be reflected by having separate names for the corresponding variables.

Reply:
I agree and that's why the one in the tests is `HUGGINGFACE_TOKEN_METATRAIN` :)